redis/src/acl.rs

```diff
@@ -162,7 +169,7 @@ impl FromRedisValue for AclInfo {
                             b"allkeys" => Ok(Rule::AllKeys),
                             b"allcommands" => Ok(Rule::AllCommands),
                             b"nopass" => Ok(Rule::NoPass),
-                            _ => Err(not_convertible_error!(flag, "Expect a valid ACL flag")),
+                            other => Ok(Rule::Other(String::from_utf8_lossy(other).to_string())),
```

Comment:
Please use `into_owned()` instead of `to_string()` here.

Reply:
Thanks! This is what I call professionalism.